tableauserverclient/server/endpoint/default_permissions_endpoint.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, parent_srv, owner_baseurl):
         self.owner_baseurl = owner_baseurl
 
     def update_default_permissions(self, resource, permissions, content_type):
-        url = '{0}/{1}/default-permissions/{2}'.format(self.owner_baseurl(), resource.id, content_type)
+        url = '{0}/{1}/default-permissions/{2}'.format(self.owner_baseurl(), resource.id, content_type + 's')
```

Comment:
I think it would be better to append the 's' to the 5 Permission.Resource enums defined in permissions.py, rather than adding it to the URL here (@t8y8 FYI). Pluralization is also needed for the populate and delete methods, so changing the enum would make be cleaner.

Reply:
There was some reason I did this here in the URLs and not in the enum, I don't remember exactly why, but I think it's because the enum gets used elsewhere where you don't want the `s`

Reply:
Not sure if it is being used elsewhere currently, but yeah it could be in the future. Within the domain of permissions only plural resource names are used, hence the suggestion, but I'm okay with this.

Reply:
I'm ok either way, I think I lean slightly towards the 's' being part of the url logic and not the resource name itself, but not sure I care a ton.

Will this create a merge conflict with the View Permissions PR? Should that one go in first?

Reply:
I don't think it will create a merge conflict. I'm waiting on the CLA for this one.

Reply:
I rethought it, and I think I changed my mind, maybe the enum should have the plural form.

Let's open a new item on that to do them all at once though

Reply:
Sorry to keep going back and forth on this, but looking more into this, making the enums plural will require more changes than I originally thought. Especially with the _set_default_permissions() method in project_item.py that sets fields by their name. 

I'll resolve this conversation. This PR is good to be merged whenever the CLA has been signed 🚀